flags/14.0/vtctld.txt

```diff
@@ -336,26 +170,16 @@ max_rate_approach_threshold: 0.9
       --vtgate_grpc_crl string                                           the server crl to use to validate server certificates when connecting
       --vtgate_grpc_key string                                           the key to use to connect
       --vtgate_grpc_server_name string                                   the server name to use to validate server certificate
-      --vtgate_protocol string                                           how to talk to vtgate (default grpc)
-      --vtworker_client_grpc_ca string                                   (DEPRECATED) the server ca to use to validate servers when connecting
-      --vtworker_client_grpc_cert string                                 (DEPRECATED) the cert to use to connect
-      --vtworker_client_grpc_crl string                                  (DEPRECATED) the server crl to use to validate server certificates when connecting
-      --vtworker_client_grpc_key string                                  (DEPRECATED) the key to use to connect
-      --vtworker_client_grpc_server_name string                          (DEPRECATED) the server name to use to validate server certificate
-      --vtworker_client_protocol string                                  (DEPRECATED) the protocol to use to talk to the vtworker server (default grpc)
-      --wait_for_drain_sleep_rdonly duration                             (DEPRECATED) time to wait before shutting the query service on old RDONLY tablets during MigrateServedTypes (default 5s)
-      --wait_for_drain_sleep_replica duration                            (DEPRECATED) time to wait before shutting the query service on old REPLICA tablets during MigrateServedTypes (default 15s)
-      --watch_replication_stream                                         When enabled, vttablet will stream the MySQL replication stream from the local server, and use it to update schema when it sees a DDL.
       --web_dir string                                                   NOT USED, here for backward compatibility
       --web_dir2 string                                                  NOT USED, here for backward compatibility
-      --workflow_manager_disable value                                   comma separated list of workflow types to disable
+      --workflow_manager_disable strings                                 comma separated list of workflow types to disable
       --workflow_manager_init                                            Initialize the workflow manager in this vtctld instance.
       --workflow_manager_use_election                                    if specified, will use a topology server-based master election to ensure only one workflow manager is active at a time.
```

Comment:
I wonder if we should deprecate these?
vtctld UI is going away, that had workflows.
We probably should also deprecate the vtctl Workflow commands.

Reply:
fixed